### PR TITLE
Make the python dependency more lenient

### PR DESCRIPTION
**Pull request by petemounce**
Based on [this comment](https://github.com/bazelbuild/bazel/issues/2027#issuecomment-262229811), allow python3 to be used. cc @meteorcloudy.

### CODE REVIEW

**Comment by bazel-io** [comment]
Can one of the admins verify this patch?

**Comment by meteorcloudy** [comment]
LGTM, thanks!

**Comment by petemounce** [comment]
Cool. I'll publish 0.4.1 RCs built from this, unless you can think of a reason not to.

**Comment by petemounce** [comment]
Couple more changes to the chocolatey build/test scripts incoming.

**Comment by petemounce** [comment]
@meteorcloudy Please take another look?

File: scripts/packages/chocolatey/bazel.nuspec.template [hunk 1]
```diff
@@ -77,7 +77,7 @@ Supply like `--params="/option:value ..."` ([see docs for --params](https://gith
     <dependencies>
       <dependency id="jdk8" version="[8.0.102,)"/>
       <dependency id="msys2" version="[20160719.1,)"/>
-      <dependency id="python2" version="[2.7.11,3.0)"/>
+      <dependency id="python" version="[2.7.11,)"/>
```
Comment (ferventcoder) [inline scripts/packages/chocolatey/bazel.nuspec.template]: @petemounce this is effectively the same as `version="2.7.11"` - it just means 2.7.11 is the minimum version that can be installed.

**Pull request closed without merging**